packages/realtime/src/index.js

```diff
@@ -75,6 +77,15 @@ async function connectWebSocket(
   numRetries,
   retryDelay
 ) {
+  const expectedStringAttributes = ['domain', 'token']
+  expectedStringAttributes.forEach(attribute => {
+    if (typeof config[attribute] !== 'string') {
+      throw new Error(
+        `Configuration attribute ${attribute} should be a string.`
+      )
+    }
+  })
```

Comment:
Minor: for this kind of code I usually go in a PropTypes manner.

```js
const isString = [
  str => typeof str === 'string',
  'should be a string'
]
const configTypes = {
  domain: isString,
  token: isString
}
const validate = config => obj => {
	for (const [attr, [validator, message]] of Object.items(configTypes)) {
      if (!validator(obj[attr])) {
        throw new Error('Attribute ${attribute} ${message}')
      }
    }
    return true
}
const validateConfig = validate(configTypes)
```

Reply:
I used this pattern to validate, thanks. I let you merge if it is ok.